cmd/plugin.go

```diff
@@ -96,10 +95,16 @@ var (
 
 			log.Info().Str("module", "plugin").Msgf("Install new plugin %s at %s", args[0], pluginDir)
 
-			// TODO: Handle already installed.
-			// TODO: Handle invalid repo name.
+			defaultVersion := "latest"
```

Comment:
What do you think changing variable name little bit?
The name 'defaultVersion' could be confused over the scope.

```golang
const defaultVersion = "latest"
...
...
...

pluginVersion := defaultVersion
if viper.GetString("plugin_version") != "" {
  pluginVersion = viper.GetString("plugin_version")
}
```

Reply:
Updated!!

```
 make build                                                                                                
fatal: No names found, cannot describe anything.
go build -ldflags="-X 'github.com/dsrvlabs/vatz/utils.Version=ISSUE-435-RE' -X 'github.com/dsrvlabs/vatz/utils.Commit=4348931'" -v
github.com/dsrvlabs/vatz
 dongyookang DK ⚡️   ~/ffplay/GolandProjects/xellos/vatz   ISSUE-435-RE
 make coverage
fatal: No names found, cannot describe anything.
?       github.com/dsrvlabs/vatz        [no test files]
?       github.com/dsrvlabs/vatz/manager/types  [no test files]
?       github.com/dsrvlabs/vatz/mocks  [no test files]
ok      github.com/dsrvlabs/vatz/cmd    1.125s  coverage: 21.5% of statements
ok      github.com/dsrvlabs/vatz/manager/api    0.320s  coverage: 0.0% of statements [no tests to run]
ok      github.com/dsrvlabs/vatz/manager/config 0.443s  coverage: 79.4% of statements
ok      github.com/dsrvlabs/vatz/manager/dispatcher     0.599s  coverage: 7.8% of statements
ok      github.com/dsrvlabs/vatz/manager/executor       0.746s  coverage: 68.9% of statements
ok      github.com/dsrvlabs/vatz/manager/healthcheck    0.889s  coverage: 46.5% of statements
ok      github.com/dsrvlabs/vatz/manager/plugin 6.142s  coverage: 50.6% of statements
ok      github.com/dsrvlabs/vatz/monitoring/prometheus  1.451s  coverage: 0.0% of statements
ok      github.com/dsrvlabs/vatz/rpc    2.182s  coverage: 67.2% of statements
ok      github.com/dsrvlabs/vatz/utils  1.309s  coverage: 7.1% of statements
 dongyookang DK ⚡️   ~/ffplay/GolandProjects/xellos/vatz   ISSUE-435-RE
 ./vatz init       
2023-05-24T17:39:43-05:00 INF init module=main
2023-05-24T17:39:43-05:00 INF create file default.yaml module=main
2023-05-24T17:39:43-05:00 INF home path ~/.vatz module=main
2023-05-24T17:39:43-05:00 INF Load Config default.yaml module=config
2023-05-24T17:39:43-05:00 INF Plugin dir /Users/dongyookang/.vatz module=main
2023-05-24T17:39:43-05:00 INF Create DB Instance module=db
2023-05-24T17:39:43-05:00 INF initDB /Users/dongyookang/.vatz/vatz.db module=db
2023-05-24T17:39:43-05:00 INF Remove old DB file module=db
2023-05-24T17:39:43-05:00 INF Create new DB file module=db
2023-05-24T17:39:43-05:00 INF Create DB Instance module=db
 dongyookang DK ⚡️   ~/ffplay/GolandProjects/xellos/vatz   ISSUE-435-RE
 ./vatz plugin list
2023-05-24T17:39:46-05:00 INF Load Config default.yaml module=config
2023-05-24T17:39:46-05:00 INF List plugins module=plugin
2023-05-24T17:39:46-05:00 INF Create DB Instance module=db
2023-05-24T17:39:46-05:00 INF List module=plugin
2023-05-24T17:39:46-05:00 INF List module=db
+------+------------+--------------+------------+---------+
| NAME | IS ENABLED | INSTALL DATE | REPOSITORY | VERSION |
+------+------------+--------------+------------+---------+
+------+------------+--------------+------------+---------+
 dongyookang DK ⚡️   ~/ffplay/GolandProjects/xellos/vatz   ISSUE-435-RE
 ./vatz plugin install github.com/dsrvlabs/vatz-plugin-sysutil/plugins/cpu_monitor cpu-monitor5 --version v0.0.0-20230221022244-43fae2fad0cd
2023-05-24T17:39:50-05:00 INF Load Config default.yaml module=config
2023-05-24T17:39:50-05:00 INF Install new plugin github.com/dsrvlabs/vatz-plugin-sysutil/plugins/cpu_monitor at /Users/dongyookang/.vatz. module=plugin
2023-05-24T17:39:50-05:00 INF Installing plugin version is v0.0.0-20230221022244-43fae2fad0cd. module=plugin
2023-05-24T17:39:50-05:00 INF Create DB Instance module=db
2023-05-24T17:39:50-05:00 INF Install new plugin github.com/dsrvlabs/vatz-plugin-sysutil/plugins/cpu_monitor module=plugin
2023-05-24T17:39:52-05:00 INF AddPlugin module=db
 dongyookang DK ⚡️   ~/ffplay/GolandProjects/xellos/vatz   ISSUE-435-RE
 ./vatz plugin install github.com/dsrvlabs/vatz-plugin-sysutil/plugins/cpu_monitor cpu-monitor4                                             
2023-05-24T17:40:01-05:00 INF Load Config default.yaml module=config
2023-05-24T17:40:01-05:00 INF Install new plugin github.com/dsrvlabs/vatz-plugin-sysutil/plugins/cpu_monitor at /Users/dongyookang/.vatz. module=plugin
2023-05-24T17:40:01-05:00 INF Installing plugin version is latest. module=plugin
2023-05-24T17:40:01-05:00 INF Create DB Instance module=db
2023-05-24T17:40:01-05:00 INF Install new plugin github.com/dsrvlabs/vatz-plugin-sysutil/plugins/cpu_monitor module=plugin
2023-05-24T17:40:03-05:00 INF AddPlugin module=db
 dongyookang DK ⚡️   ~/ffplay/GolandProjects/xellos/vatz   ISSUE-435-RE
 ./vatz plugin list                                                                                                                         
2023-05-24T17:40:09-05:00 INF Load Config default.yaml module=config
2023-05-24T17:40:09-05:00 INF List plugins module=plugin
2023-05-24T17:40:09-05:00 INF Create DB Instance module=db
2023-05-24T17:40:09-05:00 INF List module=plugin
2023-05-24T17:40:09-05:00 INF List module=db
+--------------+------------+---------------------+-------------------------------------------------------------+------------------------------------+
| NAME         | IS ENABLED | INSTALL DATE        | REPOSITORY                                                  | VERSION                            |
+--------------+------------+---------------------+-------------------------------------------------------------+------------------------------------+
| cpu-monitor5 | true       | 2023-05-24 17:39:52 | github.com/dsrvlabs/vatz-plugin-sysutil/plugins/cpu_monitor | v0.0.0-20230221022244-43fae2fad0cd |
| cpu-monitor4 | true       | 2023-05-24 17:40:03 | github.com/dsrvlabs/vatz-plugin-sysutil/plugins/cpu_monitor | latest                             |
+--------------+------------+---------------------+-------------------------------------------------------------+------------------------------------+

```